fixtures/app/extensions/pos-ui-extension/src/index.jsx

```diff
@@ -0,0 +1,28 @@
+import { extend, Text } from "@shopify/retail-ui-extensions";
+
+extend('Retail::SmartGrid::Tile', (root, api) => {
```

Comment:
did you intend to modify the pos template? this file is the fixtures/app that we use for testing. Is good to keep it updated, just wondering if you also wanted to update the template

Reply:
@isaacroldan: Initially It was not intended to add `pos_ui_extension` into fixtures. However, adding `pos_ui_extension` in fixtures helped me to quickly start an extension and investigate the root cause of the issue we were having. Thats why I kept it here so it can be used for future testing/investigations.